staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/marshal_cbor.go

```diff
@@ -0,0 +1,26 @@
+package v1
```

Comment:
Nit: I think it would be fair to add these marshaler implementations and tests to the existing source files, since both implementations need to be maintained in parallel.